course/steps/03-clients-analytics/en.md

```diff
@@ -92,9 +100,9 @@ In this step, we will implement the Anaylitcs client. So,
     }
   ```
 
-6. Now let's test it! It's possible to use Postman to send a `GET` request to the following route:
+7. Now let's test it! It's possible to use Postman to send a `GET` request to the following route:
 
-   `{your workspace}--appliancetheme.myvtex.com/_v/app/analytics/realTime`
+   `{your workspace}--{your account}.myvtex.com/_v/app/analytics/realTime`
```

Comment:
In this case, we use `appliancetheme` as the account, because it's the account that we give students the necessary permissions.